src/common/utxobased/network/BlockBook.ts

```diff
@@ -1,9 +1,16 @@
+import { Cleaner } from 'cleaners'
 import { EdgeLog, EdgeTransaction } from 'edge-core-js'
 
 import { EngineEmitter, EngineEvent } from '../../plugin/makeEngineEmitter'
 import {
   addressMessage,
   addressUtxosMessage,
+  asAddressUtxosCleaner,
+  asINewBlockResponseCleaner,
+  asINewTransactionResponseCleaner,
+  asIServerInfoCleaner,
+  asITransactionBroadcastResponseCleaner,
+  asITransactionCleaner,
```

Comment:
Remove the word `Cleaner` again here. This way the cleaner for `ITransactionCleaner` is simply `asITransactionCleaner`

Reply:
Tip: if you're using vscode you can secondary/right click on the identifier and choose Rename Symbol to easily refactor your code when it comes to name changes. I made this into a keyboard short-cut.


<img width="319" alt="Screen Shot 2021-06-25 at 12 03 40" src="https://user-images.githubusercontent.com/517469/123473453-6f66d200-d5ad-11eb-8dce-7275a80df407.png">

Reply:
I saved it for last so don't pollute the changes with renames.